### DISCUSSION TITLE
Code Folder?

### DISCUSSION THREAD

**Discussion by Morbidous** [MNES-Emulator/MNES#19] [Q&A] [answered]
In my own projects I put my C# code into a root `Code` folder (that is not in the namespace). I know that in some languages (Java) I think they call this folder `src`. I am tempted to do this but it would involve a move of basically every source file. I am eager to do this but am not sure what you think about it or if you would care what the name is.

(Sometimes I will have some other folders along with Code like Resources or Data (which just has some sql files in it). Some kinds of project like ASP.NET you can't always have super customized folder layouts since everything has to be in pre-determined places.)

I think I will at least start doing this for new projects that I add. You can let me know what you think of this.

**Comment by jamieyello** [comment, marked as answer]
Sure that sounds good, I've seen people put code into src files before, seems pretty common practice. I suppose it keeps it away from the other GitHub junk.

**Reply by Morbidous** [reply]
k thanks. I'm now "terrified" of doing this since it will modify all source files at once which will clobber whatever your working on.

**Reply by jamieyello** [reply]
Don't worry, anything that breaks won't be too hard to fix pretty quickly.
